source/develop/release-management/features/metrics/metrics-store-installation.html.md

```diff
@@ -19,13 +19,18 @@ The default name for ovirt_env_name is "engine".
    Use the following convention: Only include alphanumeric characters and hyphens ( "-" ). Name cannot begin with a hyphen or a number,
    or end with a hyphen. Maximum of 49 characters. Wildcard patterns (e.g. ovirt-metrics*) cannot be used.
 
-During the installation in the step [Configuring mux](https://github.com/ViaQ/Main/blob/master/README-install.md#configuring-mux) , use the following namespaces to create the indexes:
+During the installation in the step [Customizing vars.yaml](https://github.com/ViaQ/Main/blob/master/README-install.md#customizing-varsyaml) , use the following namespaces to create the indexes:
```

Comment:
Suggest:
"During installation, in the step..."

Also:
remove extra space: "varsyaml) , use the"